src/irmin/type/type.mli

```diff
@@ -310,6 +313,10 @@ val to_string : 'a t -> 'a -> string
 val of_string : 'a t -> 'a of_string
 (** [of_string t] parses values of type [t]. *)
 
+val boxed : 'a t -> 'a t
```

Comment:
This `val` probably belongs above line 284 "Generic Operations", since it's a constructor.